mailboxes/lang/de

```diff
@@ -77,26 +78,26 @@ mail_tofilter=Filter nach Empfänger erstellen
 mail_subfilter=Filter nach Betreff erstellen
 mail_unknown=Unbekannt
 
-mail_sign=Unterschreibe mit GnuPG-Schlüssel
+mail_sign=Mit GnuPG-Schlüssel unterschreiben
 mail_nosign=&lt;Nicht signieren&gt;
 mail_crypt=GnuPG-Verschlüsselung für:
 mail_nocrypt=&lt;Nicht verschlüsseln&gt;
 mail_samecrypt=&lt;Schlüssel der Zieladressen&gt;
 mail_addresses=Adressbuch bearbeiten
 mail_folders=Ordner bearbeiten
-mail_err=Beim Auflisten der Mails in diesem Ordner ist ein Fehler aufgetreten : $1
+mail_err=Beim Auflisten der Mails in diesem Ordner ist ein Fehler aufgetreten: $1
 mail_loginheader=POP3-Server-Login
-mail_loginheader2=IMAP Server Login
-mail_logindesc=Sie müssen Benutzername und Passwort eingeben,<br>um Ihren Posteingang auf dem Mailserver $1 einsehen zu können.
+mail_loginheader2=IMAP-Server-Login
+mail_logindesc=Benutzername und Passwort eingeben, um den Posteingang auf dem Mailserver $1 einsehen zu können.
```

Comment:
Same question here as [before](https://github.com/webmin/webmin/pull/2266/files#r1741879243).

Reply:
> In English language there's no real rule when to use spaces, dashes, or join two words (think of Wonder Woman, Spider-Man, Batman).

Oh, I'm perfectly fine with the change related to dashes. Danke!

However, removing `<br>` is not okay.

> Main reason for the other change was the avoidance of (non-)formal language. In German, we use two different forms. Some prefer the informal version when dealing with software and apps, others don't. Hence, the suggested change.

Please see https://github.com/webmin/webmin/pull/2266#discussion_r1741905136.